paderbox/array/interval/core.py

```diff
@@ -358,6 +358,42 @@ def _intervals_as_str(self):
         i_str = ', '.join(i_str)
         return i_str
 
+    @property
+    def as_serializable(self):
+        """
+        Exports intervals and length of ArrayInterval to a serializable object.
+        Reverting 
+        Allows easy export of ArrayIntervals, e.g. into .json format. 
+        >>> from IPython.lib.pretty import pprint
+        >>> from paderbox.array.interval.core import ArrayInterval
+        >>> ai = ArrayInterval.from_str('1:4, 5:20, 21:25', shape=50)
+        >>> ai
+        ArrayInterval("1:4, 5:20, 21:25", shape=(50,))
+        >>> ai.as_serializable
+        ('1:4, 5:20, 21:25', (50,))
+        """
+        assert self.inverse_mode is False, 'Export of intervals as tuple is only valid for normal mode, not inverse mode!' 
+        return self._intervals_as_str, self.shape
+
+    @staticmethod
+    def from_serializable(array):
```

Comment:
I think the name `array` is confusing here. I would either call it `serializable` or give it a non-informative name like `obj`.